src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -0,0 +1,227 @@
+
+## Notification integration detail [#detail]
+
+Read about more about each of our specific notification integrations.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="email"
+    title="Email"
+  >
+  Send email notifications to users with or without New Relic accounts. You don't need to create new destinations for emails, as they're automatically added from email addresses in the event template.
+
+## Configure the event template [#configure-event-template]
+
+Users with New Relic accounts can be found via autocomplete when searching for first or last name.
+
+To add a user without a New Relic account or email distribution list, add the full email address.
+
+## Send a test notification [#email-test]
+
+You can send a test notification with a pre-defined example title and body to the emails you define.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="jira"
+    title="Jira"
+  >
+
+Send notifications to users of Atlassian Jira.
+
+## Permissions [#permissions]
+
+The required permissions from the Jira `API-Token` are `create`, `edit`, and `close` tickets.
+
+To enable the two-way sync toggle, the provided Jira `API-Key` needs  an `Admin` role.
+
+## Sett up a Jira destination [#set-up-jira]
+
+Create Jira issues, then enable Jira and New Relic to share updates and stay synced.
+
+To create a Jira destination, enter the following information:
+
+* :**Destination Name::** Custom name to identify the destination
+* :**Jira account endpoint::** the URL of the destination
+* :**User-name:** this will be the email address of the user making the connection
+* :**API Token:** generated from your Atlassian account
+
+<Callout variant="important">
+  New Relic currently supports company-managed projects.
+</Callout>
+
+Before saving the destination, we recommend you test the connection via the **test connection** button.
+
+## Two-way sync [#jira-two-way]
+
+To enable two-way sync between Jira and New Relic One, enable the `two-way integration` toggle.
+
+When enabled, a Jira webhook would be created in your Jira account at a later stage, for the selected project (see ‘customize a message template’ below). The webhook would contain access details to Newrelic (URL and Newrelic API-KEY).
+
+By default, any update to a Jira issue created by New Relic One will sync back to New Relic One. After enabling two-way integration, when Jira issue changes to closed the corresponding workflow issue will change to Closed.
+
+## Configure the message template [#configure-jira-message-template]
+
+Upon successful connection, Jira fields are fetched from your account and automatically mapped to your Jira instance. 
+
+To help you get started, recommended fields and values are automatically presented. You can add or remove fields and change the values to customize the notification to your needs.
```

Comment:
"we present recommended fields"? It's a tricky place to inject active voice, so I'll point it out, but it doesn't have to change.